src/components/verify/verify-page.tsx

```diff
@@ -10,7 +10,7 @@ import { DropZone } from "./dropzone";
 
 const NETWORK_NAME = process.env.REACT_APP_NETWORK_NAME || "ropsten";
 
-enum Status {
+export enum Status {
```

Comment:
I think this create a cycle
- navigation bar import verify page
- verify page import navigation bar

Let's extract this out into another file (general types?)